pkg/scheduler/factory.go

```diff
@@ -127,6 +127,13 @@ func (c *Configurator) create() (*Scheduler, error) {
 				Name: noderesources.FitName,
 				Args: &schedulerapi.NodeResourcesFitArgs{
 					IgnoredResources: ignoredExtendedResources,
+					ScoringStrategy: &schedulerapi.ScoringStrategy{
```

Comment:
This is not correct. If a plugin config was set, then this will simply override the user parameters. The solution might not be straightforward, let me think about it.

Reply:
So one solution is to move the defaulting logic for non-existent plugin configs to component config defaulting, and then here we lookup the plugin config and only update the IgnoredResources field. We need https://github.com/kubernetes/kubernetes/pull/99597 to merge first.

Reply:
ok, and ci fail is expect.

Reply:
yeah, we need to get the defaulting right first. can you resolve all open comments and leave this one only please so we know where we are.

Reply:
Sure.

Reply:
I think we can resolve this one now, the tests should pass once you rebase.